### PR TITLE
MMM: scale contributions to original scale and allow custom colors

### PR DESCRIPTION
**Pull request by juanitorduz**
Builds upon https://github.com/pymc-labs/pymc-marketing/pull/247

- [x] Scale contributions to the original scale.
- [x] Allows custom colors.

----
:books: Documentation preview :books:: https://pymc-marketing--251.org.readthedocs.build/en/251/

### CODE REVIEW

**Comment by juanitorduz** [comment]
What do you think @Garve ? :)

**Comment by Garve** [comment]
Heya! Should we also introduce a flag for the original scale? Like in the other plot method.

File: tests/mmm/test_base.py [hunk 1]
```diff
@@ -210,7 +210,11 @@ def toy_preprocessing(self, data):
             ("plot_grouped_contribution_breakdown_over_time", {}),
             (
                 "plot_grouped_contribution_breakdown_over_time",
-                {"stack_groups": {"controls": ["control_1"]}},
+                {
```
Comment (ricardoV94) [inline tests/mmm/test_base.py:213]: Not for this PR, but I wonder if we should test the actual expected plots with something like https://matplotlib.org/stable/api/testing_api.html#module-matplotlib.testing

At least we will know if they accidentally change. Not super convinced.

**Review by ricardoV94 — APPROVED** [review]

File: tests/mmm/test_base.py (hunk 1 above)
Comment (juanitorduz) [reply tests/mmm/test_base.py:213]: wow! I did not know about this! I'll take a look and create an issue

**Pull request merged**